.github/workflows/cmake-multi-platform.yml

```diff
@@ -45,31 +45,33 @@ jobs:
             c_compiler: cl
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
 
```

Comment:
Would make reviewing easier if whitespace change (in .github/workflows/cmake-multi-platform.yml) were in a separate commit